### PR TITLE
Migrated Food entity to SQLalchemy

### PR DESCRIPTION
**Pull request by zzzrg**
# Description

Changed Food entity to work with SQLalchemy.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.  
Provide instructions so we can **reproduce**.  
Please also list any relevant details for your test configuration

- [x] Added a unittest test_food()

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I will rebase before merging

### CODE REVIEW

File: backend/app/main/main.py [hunk 1]
```diff
@@ -70,46 +70,22 @@ def create_vendor(vendor: schemas.VendorCreate, session: Session = Depends(get_d
...
-#         "category":"str",
-#         "serving_size":"str"
-#     }
-
-#     :return: JSON response with created entry
-#     """
-#     create_food(item)
-#     return item
+@router.post("/add_food", response_model=schemas.Food)
+def food(food: schemas.FoodCreate, session: Session = Depends(get_db)):
```
Comment (TamPhilip) [inline backend/app/main/main.py:74]: Maybe make the function name more consistent so name it "create_food" (like the vendor).

File: backend/app/main/main.py [hunk 2]
```diff
@@ -70,46 +70,22 @@ def create_vendor(vendor: schemas.VendorCreate, session: Session = Depends(get_d
...
+    """
+    Adds a new vendor to the database - MealCare
```
Comment (TamPhilip) [inline backend/app/main/main.py:76]: Change the comment

Comment (TamPhilip) [inline backend/app/main/main.py:76]: Change the comment to adds a new "food" or "food tray"

File: backend/app/main/main.py (hunk 1 above)
Comment (TamPhilip) [inline backend/app/main/main.py:74]: Make the naming of the function more consistent like the vendor so "create_food" instead of only food

Comment (zzzrg) [reply backend/app/main/main.py:74]: oops sorry good catch

**Pull request merged**